Content.Client/PowerCell/PowerCellSystem.cs

```diff
@@ -48,17 +48,14 @@ private void OnPowerCellVisualsChange(EntityUid uid, PowerCellVisualsComponent c
         if (!_sprite.LayerExists((uid, args.Sprite), PowerCellVisualLayers.Unshaded))
             return;
 
-        if (_appearance.TryGetData<byte>(uid, PowerCellVisuals.ChargeLevel, out var level, args.Component))
-        {
-            if (level == 0)
-            {
-                _sprite.LayerSetVisible((uid, args.Sprite), PowerCellVisualLayers.Unshaded, false);
-                return;
-            }
+        if (!_appearance.TryGetData<byte>(uid, PowerCellVisuals.ChargeLevel, out var level, args.Component))
```

Comment:
While you're at it, please fix this check: the visualizer should default this value, not bail entirely.